ccl/changefeedccl/cdcevent: TestEventColumnOrderingWithSchemaChanges failed

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17958424?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17958424?buildTab=artifacts#/) on master @ [0f2422bac446d1d297cec730ec7817aef46bd2be](https://github.com/cockroachdb/cockroach/commits/0f2422bac446d1d297cec730ec7817aef46bd2be):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/63f86ed7e3c77b1ceae8fd1a6c3d722c/logTestEventColumnOrderingWithSchemaChanges2824618864
    test_log_scope.go:76: use -show-logs to present logs inline
    event_test.go:819: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/63f86ed7e3c77b1ceae8fd1a6c3d722c/logTestEventColumnOrderingWithSchemaChanges2824618864
--- FAIL: TestEventColumnOrderingWithSchemaChanges (12.04s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite (0.00s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite (11.17s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite (0.00s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite (0.00s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-45004

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/c2347dde-4749-4bfb-a318-42b28c8ea4a5?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [7d48198a57f014a8828194b90098699f70f0695a](https://github.com/cockroachdb/cockroach/commits/7d48198a57f014a8828194b90098699f70f0695a):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges2083255564
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges2083255564
--- FAIL: TestEventColumnOrderingWithSchemaChanges (20.54s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite (12.64s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite (0.01s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17969815?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17969815?buildTab=artifacts#/) on master @ [f791ddf89427dd526aef1eec5c14abe4a1bb5f79](https://github.com/cockroachdb/cockroach/commits/f791ddf89427dd526aef1eec5c14abe4a1bb5f79):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/63f86ed7e3c77b1ceae8fd1a6c3d722c/logTestEventColumnOrderingWithSchemaChanges989257293
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    event_test.go:819: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/63f86ed7e3c77b1ceae8fd1a6c3d722c/logTestEventColumnOrderingWithSchemaChanges989257293
--- FAIL: TestEventColumnOrderingWithSchemaChanges (14.19s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite (10.88s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (0.00s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/10a83506-7b7a-4ec9-91b3-46d58825d77e?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [bcc993d796d03664604bf695e38fd5644d0bc952](https://github.com/cockroachdb/cockroach/commits/bcc993d796d03664604bf695e38fd5644d0bc952):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges3241795533
    test_log_scope.go:76: use -show-logs to present logs inline
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges3241795533
--- FAIL: TestEventColumnOrderingWithSchemaChanges (23.11s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (12.60s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/5c9b0714-e971-432c-a162-bc020d586966?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [b1474fa887606008960634b571cf4501efb6281b](https://github.com/cockroachdb/cockroach/commits/b1474fa887606008960634b571cf4501efb6281b):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges3332482289
    test_log_scope.go:76: use -show-logs to present logs inline
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges3332482289
--- FAIL: TestEventColumnOrderingWithSchemaChanges (23.55s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (12.63s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/0b76d51f-77c2-440e-bdf5-dde4e96d25c4?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [749b563a7b5957b2db487c6e8e2fdeed8630103d](https://github.com/cockroachdb/cockroach/commits/749b563a7b5957b2db487c6e8e2fdeed8630103d):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges3889296686
    test_log_scope.go:76: use -show-logs to present logs inline
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges3889296686
--- FAIL: TestEventColumnOrderingWithSchemaChanges (15.71s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (0.02s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite (12.82s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18004372?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18004372?buildTab=artifacts#/) on master @ [f7c4d5429516e0ad6275dc50a332576a5811be0c](https://github.com/cockroachdb/cockroach/commits/f7c4d5429516e0ad6275dc50a332576a5811be0c):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/63f86ed7e3c77b1ceae8fd1a6c3d722c/logTestEventColumnOrderingWithSchemaChanges3873806402
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    event_test.go:819: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/63f86ed7e3c77b1ceae8fd1a6c3d722c/logTestEventColumnOrderingWithSchemaChanges3873806402
--- FAIL: TestEventColumnOrderingWithSchemaChanges (14.95s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (10.99s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/11802f68-0204-4eef-8f3d-9a7598f7085e?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [b3fec61ca90095c664f6432af864f18e9946f8bb](https://github.com/cockroachdb/cockroach/commits/b3fec61ca90095c664f6432af864f18e9946f8bb):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges2087404881
    test_log_scope.go:76: use -show-logs to present logs inline
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges2087404881
--- FAIL: TestEventColumnOrderingWithSchemaChanges (15.51s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite (0.03s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (0.03s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite (0.03s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite (12.70s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/4f2e721e-604c-45c9-8aba-ee97798b6f04?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [44de2d379610067e14a7ebfbc92e64311f13a232](https://github.com/cockroachdb/cockroach/commits/44de2d379610067e14a7ebfbc92e64311f13a232):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges1802752388
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges1802752388
--- FAIL: TestEventColumnOrderingWithSchemaChanges (30.65s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (13.06s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/46a09723-36b8-41f3-bdc9-56d87aeaa3d2?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [844d7637f3b4dc1275e8aa05c1cf3bbb1f59f8eb](https://github.com/cockroachdb/cockroach/commits/844d7637f3b4dc1275e8aa05c1cf3bbb1f59f8eb):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges139598647
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges139598647
--- FAIL: TestEventColumnOrderingWithSchemaChanges (21.23s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite (12.77s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (0.01s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/e879b55b-74be-45c5-8361-9dc163ff2659?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [21d1123dce5401b1ec580d4156db5db075074506](https://github.com/cockroachdb/cockroach/commits/21d1123dce5401b1ec580d4156db5db075074506):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges1129777202
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges1129777202
--- FAIL: TestEventColumnOrderingWithSchemaChanges (25.12s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (12.58s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18071937?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18071937?buildTab=artifacts#/) on master @ [45aae1027610d1a3a38d63c97ac8230b126ee069](https://github.com/cockroachdb/cockroach/commits/45aae1027610d1a3a38d63c97ac8230b126ee069):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/63f86ed7e3c77b1ceae8fd1a6c3d722c/logTestEventColumnOrderingWithSchemaChanges2806573731
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    event_test.go:819: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/63f86ed7e3c77b1ceae8fd1a6c3d722c/logTestEventColumnOrderingWithSchemaChanges2806573731
--- FAIL: TestEventColumnOrderingWithSchemaChanges (11.96s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (0.00s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite (0.00s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite (10.92s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite (0.00s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite (0.01s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/73ee2d44-b3ce-4175-bff1-4c07407643b8?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [ccaed974f75032f1b908bca2d42145d21d0092ca](https://github.com/cockroachdb/cockroach/commits/ccaed974f75032f1b908bca2d42145d21d0092ca):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges4121549727
    test_log_scope.go:76: use -show-logs to present logs inline
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges4121549727
--- FAIL: TestEventColumnOrderingWithSchemaChanges (23.62s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (12.75s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/f432ddc5-d368-4391-b52a-57e690ba88f0?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [d7ea85402dc35e36c6cc35520fa91f25fd5c999d](https://github.com/cockroachdb/cockroach/commits/d7ea85402dc35e36c6cc35520fa91f25fd5c999d):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges2327323783
    test_log_scope.go:76: use -show-logs to present logs inline
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges2327323783
--- FAIL: TestEventColumnOrderingWithSchemaChanges (18.35s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite (12.35s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://tanzanite.cluster.engflow.com/invocations/default/d3637161-f95e-440e-9cac-ddd925f15ea5?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2wvY2RjZXZlbnQ6Y2RjZXZlbnRfdGVzdA==) on master @ [74e83fa45e549680ed0e1ce8917468701bea7794](https://github.com/cockroachdb/cockroach/commits/74e83fa45e549680ed0e1ce8917468701bea7794):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestEventColumnOrderingWithSchemaChanges2722100932
    test_log_scope.go:76: use -show-logs to present logs inline
    event_test.go:819: -- test log scope end --
test logs left over in: outputs.zip/logTestEventColumnOrderingWithSchemaChanges2722100932
--- FAIL: TestEventColumnOrderingWithSchemaChanges (14.88s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_no_rewrite (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_rewrite (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (0.01s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite
    event_test.go:733: error executing query="\n\t\t\t\tCREATE TABLE foo (\n\t\t\t\t\ti INT,\n\t\t\t\t\tj INT,\n\t\t\t\t\ta STRING,\n\t\t\t\t\tb STRING,\n\t\t\t\t\tc STRING,\n\t\t\t\t\td STRING AS (concat(e, '.')) VIRTUAL,\n\t\t\t\t\te STRING,\n\t\t\t\t\tPRIMARY KEY(j,i),\n\t\t\t\t\tFAMILY main (i,j,a,b),\n\t\t\t\t\tFAMILY ec (e,c)\n\t\t\t)" args=[]: pq: relation "defaultdb.public.foo" already exists
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_no_rewrite (0.02s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/main/main_cols_rewrite (12.23s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl/cdcevent.TestEventColumnOrderingWithSchemaChanges [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18154836?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18154836?buildTab=artifacts#/) on master @ [36df22df169677631b5e9cd793d8973c25fd5f71](https://github.com/cockroachdb/cockroach/commits/36df22df169677631b5e9cd793d8973c25fd5f71):


```
=== RUN   TestEventColumnOrderingWithSchemaChanges
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/63f86ed7e3c77b1ceae8fd1a6c3d722c/logTestEventColumnOrderingWithSchemaChanges2116837274
    test_log_scope.go:76: use -show-logs to present logs inline
    event_test.go:819: -- test log scope end --
--- FAIL: TestEventColumnOrderingWithSchemaChanges (15.52s)
=== RUN   TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual
    event_test.go:773: timeout reading row
    --- FAIL: TestEventColumnOrderingWithSchemaChanges/ec/ec_cols_with_virtual (10.71s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestEventColumnOrderingWithSchemaChanges.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/changefeedccl/cdcevent/BUILD.bazel
+++ b/pkg/ccl/changefeedccl/cdcevent/BUILD.bazel
@@ -78,6 +78,7 @@ go_test(
         "//pkg/sql/schemachanger/scplan",
         "//pkg/sql/sem/eval",
         "//pkg/sql/sem/tree",
+        "//pkg/sql/sqltestutils",
         "//pkg/sql/types",
         "//pkg/testutils/serverutils",
         "//pkg/testutils/skip",

--- a/pkg/ccl/changefeedccl/cdcevent/event_test.go
+++ b/pkg/ccl/changefeedccl/cdcevent/event_test.go
@@ -26,6 +26,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/schemachanger/scplan"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/eval"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/sql/sqltestutils"
 	"github.com/cockroachdb/cockroach/pkg/sql/types"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
@@ -595,7 +596,12 @@ func TestEventColumnOrderingWithSchemaChanges(t *testing.T) {
 		kvserver.RangefeedEnabled.Override(ctx, &l.ClusterSettings().SV, true)
 	}
 
+	// Be aggressive with the SCHEMA CHANGE GC job. For tests involving column
+	// rewrites, this job must run for the tests to succeed. Its execution
+	// triggers RangeFeedDeleteRange events.
 	sqlDB := sqlutils.MakeSQLRunner(db)
+	defer sqltestutils.DisableGCTTLStrictEnforcement(t, db)()
+	sqlDB.Exec(t, "SET CLUSTER SETTING jobs.registry.interval.adopt='1s'")
 
 	type decodeExpectation struct {
 		expectUnwatchedErr bool
@@ -730,8 +736,9 @@ func TestEventColumnOrderingWithSchemaChanges(t *testing.T) {
 		},
 	} {
 		t.Run(tc.testName, func(t *testing.T) {
-			sqlDB.Exec(t, `
-				CREATE TABLE foo (
+			sqlDB.ExecMultiple(t,
+				`DROP TABLE IF EXISTS foo`,
+				`CREATE TABLE foo (
 					i INT,
 					j INT,
 					a STRING,
@@ -747,6 +754,9 @@ func TestEventColumnOrderingWithSchemaChanges(t *testing.T) {
 			tableDesc := cdctest.GetHydratedTableDescriptor(t, s.ExecutorConfig(), "foo")
 			popRow, cleanup := cdctest.MakeRangeFeedValueReaderExtended(t, s.ExecutorConfig(), tableDesc)
 			defer cleanup()
+
+			_, err := sqltestutils.AddImmediateGCZoneConfig(db, tableDesc.GetID())
+			require.NoError(t, err)
 
 			targetType := jobspb.ChangefeedTargetSpecification_EACH_FAMILY
 			if tc.familyName != "" {
@@ -769,8 +779,10 @@ func TestEventColumnOrderingWithSchemaChanges(t *testing.T) {
 			require.NoError(t, err)
 
 			expectedEvents := len(tc.expectMainFamily) + len(tc.expectECFamily)
+			log.Infof(ctx, "expectedEvents: %d\n", expectedEvents)
 			for i := 0; i < expectedEvents; i++ {
 				v, deleteRange := popRow(t)
+				log.Infof(ctx, "event[%d]: v=%+v, deleteRange=%+v", i, v, deleteRange)
 
 				if deleteRange != nil {
 					// Should not see a RangeFeedValue and a RangeFeedDeleteRange

